src/test/java/teammates/test/cases/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -121,6 +121,9 @@ private void testSubmitAction() throws Exception {
         submitPage = loginToStudentFeedbackSubmitPage("Alice", "Open Session");
         submitPage.waitForPageToLoad();
 
+        boolean confirmationEmailBox = submitPage.isConfirmationEmailBoxTicked();
+        assertTrue(confirmationEmailBox);
```

Comment:
These two lines 124 and 125 can be combined to `assertTrue(submitPage.isConfirmationEmailBoxTicked());`

Reply:
Made this change :smile: